macos-install_dependencies.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Install Xcode Command Line Tools
+xcode-select --install
+
+# Install Homebrew
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+# Install Rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+# Install CMake
+brew install cmake
+
+# Install FFmpeg
+brew install ffmpeg
+
+# Install Opus
+brew install opus
```

Comment:
I havent checked but dont brew allow multiple dependencies to be installed at once or one at a time?

Reply:
it does, i can update for that :joy:

Reply:
done 🎉